### PR TITLE
Updated README.md

### LINKED ISSUE #5: [Feature Request]: Readme Update
### Preflight Checklist

- [X] I have read the Contribution.md for this project.
- [X] I agree to follow the Code of Conduct that this project adheres to.
- [X] I have searched the issue for a feature request that matches the one I want to file, without success.

### Problem Description

Proper Readme is not available.

### Proposed Solution

Updating Readme file with proper description and icons

### Alternatives Considered

NA

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by Sukriti-m**
# Description:

Updated README.md. Added  proper description and icons as well.

Fixes #5 

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

# Screenshots / Video:

https://user-images.githubusercontent.com/93239528/162127625-2706f4b5-6912-40a9-a3fd-7e5efb9a8bc6.mp4

@Adi-11 Please review this PR.

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1 +1,2 @@
-REACT_APP_API_BASE_URL=XXXXX
\ No newline at end of file
+REACT_APP_API_BASE_URL=XXXXX
+REACT_APP_API_BASE_URL_LOCAL=http://localhost:4000
```
Comment (Adi-11) [inline .env.sample]: `REACT_APP_API_BASE_URL_LOCAL=http://localhost:4000 `

Revert this change.

Comment (Adi-11) [reply .env.sample]: @Sukriti-m

Comment (Sukriti-m) [reply .env.sample]: Hey @Adi-11 
I have made the required changes. Please review it.

**Review by Adi-11 — APPROVED** [review]

**Comment by Sukriti-m** [comment]
@Adi-11  Please add the labels as well.

**Comment by Adi-11** [comment]
@Sukriti-m yes added

**Comment by Sukriti-m** [comment]
@Adi-11 Thank you 😃
